### ISSUE TITLE
[FEAT] :sparkles: button to delete entries from tables

### ISSUE THREAD

**Issue by aayushchugh** [keizerworks/invoicen#24]
### Description

We need to add a button to delete entries from tables

<img width="1154" alt="image" src="https://github.com/user-attachments/assets/e9db6e07-a5ad-4888-b4c7-9fe5aa284a3b">

<img width="1123" alt="image" src="https://github.com/user-attachments/assets/7fb6c453-b991-4760-822f-01e1c8219688">

### Are you willing to submit PR?

No

**Comment by pavanydg** [comment]
@aayushchugh can i work on this?

**Comment by aayushchugh** [comment]
sure! you can work on this after #26 is merged

**Comment by aayushchugh** [comment]
@pavanydg You can start working on this now

**Comment by sanatan-dive** [comment]
@aayushchugh can i work on it?

**Comment by aayushchugh** [comment]
@pavanydg is already working on it, he has implemented the logic. Now we just need some UI changes. Maybe you can collaborate with him

**Comment by sanatan-dive** [comment]
@aayushchugh yeah i have already done those ui changes which u want but the logic is different it works but.
all is left just pulling and merging
![oie_jQ59Nq5aN4KQ](https://github.com/user-attachments/assets/71afa5e3-7442-4a2d-966d-86dc57dd33d3)

**Comment by aayushchugh** [comment]
Niceee, raise a PR and I'll review.
